srcpkgs/zxing-cpp/template

```diff
@@ -1,10 +1,11 @@
 # Template file for 'zxing-cpp'
 pkgname=zxing-cpp
 version=2.3.0
-revision=1
+revision=2
 build_style=cmake
-configure_args="-DBUILD_EXAMPLES=OFF -DBUILD_BLACKBOX_TESTS=OFF
+configure_args="-DBUILD_EXAMPLES=ON -DBUILD_BLACKBOX_TESTS=OFF
  -DBUILD_DEPENDENCIES=LOCAL"
+hostmakedepends="pkg-config stb"
```

Comment:
even if `stb` works in `hostmakedepends` its probably better to put it in `makedepends`, since the target compiler uses that sysroot when cross compiling.

Reply:
Done.